docs/index.adoc

```diff
@@ -5,7 +5,7 @@ include::variables.adoc[]
 
 == Introduction
 
-Explorer is an enterprise search application that supports collecting, and searching across data from different sources. 
+Explorer is an enterprise search application that supports collecting, and searching across data from different sources. You need a license to run the application. 
```

Comment:
You do no need a license to run the application. You need a license to have more than 3 collections, synonyms or more than 1 interface I think.

Reply:
https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/Collections.jsx#L162

Reply:
https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/thesaurus/NewOrEditThesaurus.jsx#L65

Reply:
You are allowed 2 interfaces https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/interfaces/NewOrEditInterfaceModal.jsx#L63